himl/main.py

```diff
@@ -63,6 +75,8 @@ def get_parser(parser=None):
                             help='the working directory')
         parser.add_argument('--multi-line-string', action='store_true',
                             help='will overwrite the global yaml dumper to use block style')
```

Comment:
Use an enum choice here please: 
https://stackoverflow.com/a/46385352/649647

Reply:
Thanks. It makes sense to put allowable values (append/override/prepend/append_unique) as enum. Just a clarification.. As the new argument --merge-list-strategy takes two forms of format, 

1. --merge-list-strategy default append/override/prepend/append_unique (predefined merge strategies deepmerge provides)
2. --merge-list-strategy user_defined_module user_defined_function (user defined)

The example you gave will not cover both scenarios. Do you suggest to just define an enum for those predefined strategies and just check them during parsing, or should we split merge-list-strategy argument to two separate arguments, one to take deepmerge defined strategies, one to custom strategies? for examp,e --merge-list-default-strategy  and merge-list-custom-strategy ?

Thanks.

Reply:
@amuraru  pushed a PR for implementation #1, defined an enum for 4 default merge strategies and validate passed in value, and throw a meaningful error msg. 
plz let me know if #2 implementation makes more sense and it's a better alternative.

Reply:
see my comment above - 2# doesn't make sense to me here